index.js

```diff
@@ -71,6 +63,7 @@ module.exports = {
     'import/first': 'error',
     'import/newline-after-import': 'error',
     'import/no-duplicates': 'error',
+    'prettier/prettier': 'error',
```

Comment:
Moved this line to make it more visible as I had trouble seeing it the first time...